src/main/java/jenkins/scm/api/SCMHead.java

```diff
@@ -207,7 +210,8 @@ public String toString() {
     @Deprecated
     @CheckForNull
     public <T extends Action> T getAction(@NonNull Class<T> type) {
-        return null;
+        // TODO should be deleted but old versions of blue ocean api rely on this
+        return ChangeRequestAction.class.equals(type) ? (T)new ChangeRequestAction(){} : null;
```

Comment:
No. We are not implementing these methods. I would much rather that blue ocean fixes this asap rather than leave them with something that is built on hacks. There is no route from an SCMHead to the persisted actions, so all we can return here is a dummy... which was why I originally said _just return null_. Happy to go back to that, not happy to do any more than this